splash/qtrender_lua.py

```diff
@@ -533,7 +536,11 @@ def delete_cookies(self, name=None, url=None):
     @command()
     def add_cookie(self, name, value, path=None, domain=None, expires=None,
                    httpOnly=None, secure=None):
-        cookie = dict(name=name, value=value)
+        name = name
+        value = value
+        cookie = dict()
+        cookie['name'] = name
+        cookie['value'] = value
```

Comment:
I made the change when I was using bytes as keys. Now that we're using unicode keys this is no longer required. I'll fix it.